security/pkg/nodeagent/secretfetcher/secretfetcher.go

```diff
@@ -443,6 +450,23 @@ func (sf *SecretFetcher) FindIngressGatewaySecret(key string) (secret model.Secr
 	val, exist := sf.secrets.Load(key)
 	secretFetcherLog.Debugf("load secret %s from secret fetcher: %v", key, exist)
 	if !exist {
+		// In some case, we might not get the secret from watch. Try to directly obtain the secret from API call
+		// as a remedy. If we can find a secret, add it to the cache
+		if sf.coreV1 != nil {
+			if secret, err := sf.coreV1.Secrets(sf.secretNamespace).Get(key, metav1.GetOptions{}); err == nil {
+				log.Infof("Find secret %s by direct api call", key)
```

Comment:
Please use secretFetcherLog instead of log.

Reply:
Thanks, will fix.